src/cosalib/aws.py

```diff
@@ -61,30 +61,31 @@ def aws_run_ore_replicate(build, args):
         'aws', 'copy-image', '--image',
         source_image, '--region', source_region
     ])
+    ore_args.extend(region_list)
+    print("+ {}".format(subprocess.list2cmdline(ore_args)))
 
-    upload_failed_in_region = None
-    for upload_region in region_list:
-        region_ore_args = ore_args.copy() + [upload_region]
-        print("+ {}".format(subprocess.list2cmdline(region_ore_args)))
-        try:
-            ore_data = json.loads(subprocess.check_output(region_ore_args))
-        except subprocess.CalledProcessError:
-            upload_failed_in_region = upload_region
-            break
-        # This matches the Container Linux schema:
-        # https://stable.release.core-os.net/amd64-usr/current/coreos_production_ami_all.json
-        ami_data = [{'name': name,
-                     'hvm': vals['ami'],
-                     'snapshot': vals['snapshot']}
-                    for name, vals in ore_data.items()]
-        buildmeta['amis'].extend(ami_data)
-        # Record the AMI's that have been replicated as they happen.
-        # When re-running the replication, we don't want to be lose
-        # what has been done.
-        build.meta_write()
-
-    if upload_failed_in_region is not None:
-        raise Exception(f"Upload failed in {upload_failed_in_region} region")
+    try:
+        ore_data = subprocess.check_output(ore_args).decode('utf-8')
```

Comment:
This is more commonly spelled
`subprocess.check_output(ore_args, encoding='utf-8')` I think.